### PR TITLE
Changes as suggetsed for chain of title icon

### PR DESCRIPTION
**Pull request by govindbrickred**
suggestion as suggested by ken

### CODE REVIEW

File: Inview.Epi.EpiFund.Web/Views/Admin/ManageHoldingCompanies.cshtml [hunk 1]
```diff
@@ -256,17 +254,33 @@
...
         //alert(PortfolioId);
         /*show model popup
          bind data as per the assets list*/
         $.ajax({
             url: "@Url.Action("GetAssetsbyPortfolioId", "Admin")",
             data: { portfolioId: PortfolioId },
             success: function (data) {
-                $("#divAssetViewPortfolio").html(data);
-                $("#portfholioAssets").modal();
+                debugger;
```
Comment (ShikharBR) [inline Inview.Epi.EpiFund.Web/Views/Admin/ManageHoldingCompanies.cshtml]: Please remove debugger

**Review by ShikharBR — COMMENTED** [review]
Please check review comment

**Review by ShrutiSaini-BRS — APPROVED** [review]
Approved

**Review by ShrutiSaini-BRS — APPROVED** [review]
Approved

**Review by ShrutiSaini-BRS — APPROVED** [review]

**Pull request merged**